src/shared/bg_public.h

```diff
@@ -445,7 +445,8 @@ enum persEnum_t
 #define PS_SPRINTTOGGLE       BIT(3)
 
 // entityState_t->eFlags
-// notice that some flags are overlapped, so their meaning depends on context
+// PLEASE DO NOT MAKE THEM OVERLAP FOR NOTHING:
```

Comment:
What's our policy for overlapping flags? Can you document it here too?

Reply:
> What's our policy for overlapping flags?

I don't think we have one, but the idea of the commit is that we don't *need* to have one so we better don't, as it avoids possible bugs

Reply:
Yes, but that's not scalable in the long term. As we add more features, we will end up with overlapping flags.

I was hoping we could come up with one right now and encode it as you are touching this file.

My idea for a policy is something like:

Every time you add an overlapping flag you note what it overlaps with and why this is safe.

Reply:
Crafting a policy sounds good, but also I don't feel like I have a strong opinion on the matter.

Reply:
Then let's add this to the code if you don't care and merge it?

Reply:
wfm